libs/build_proj.sh

```diff
@@ -41,7 +41,7 @@ cd ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
 software=$name-$version
 URL="https://download.osgeo.org/proj/$software.tar.gz"
 
-[[ -d $software ]] || ( $WGET $URL; tar -xf $software.tar.gz )
+[[ -d $software ]] || ( $WGET --no-check-certificate $URL; tar -xf $software.tar.gz )
```

Comment:
Curious about what this certificate check is for?

Reply:
Is that on Orion? I put in a ticket about that a few months ago.

Reply:
perhaps an easier solution is to define
```
export WGET="wget --no-check-certificate
```
in your `config.sh`